### PR TITLE
Add auth config for uma-connect-button

### PR DESCRIPTION
**Pull request by bsiaotickchong**

### CODE REVIEW

**Comment by bsiaotickchong** [comment]
* **#136** <a href="https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/136?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bsiaotickchong and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/html/index.html [hunk 1]
```diff
@@ -21,6 +21,15 @@
     <h1>UMA Auth Client html demo</h1>
     <p>Click the UMA Connect button to get started.</p>
     <script src="../../packages/uma-auth-client/dist/uma-auth-client-web-components.umd.cjs"></script>
-    <uma-connect-button></uma-connect-button>
+    <uma-connect-button
+      nostr-pubkey="npub1scmpzl2ehnrtnhu289d9rfrwprau9z6ka0pmuhz6czj2ae5rpuhs2l4j9d"
+      nostr-relay="wss://nos.lol"
+      redirect-uri="http://localhost:3001"
+      required-commands="pay_invoice,get_balance"
```
Comment (jklein24) [inline examples/html/index.html:28]: Should this be a list of commands instead from a typing standpoint or does it always have to be a string with webcomponents?

File: examples/html/index.html [hunk 2]
```diff
@@ -21,6 +21,15 @@
     <h1>UMA Auth Client html demo</h1>
     <p>Click the UMA Connect button to get started.</p>
     <script src="../../packages/uma-auth-client/dist/uma-auth-client-web-components.umd.cjs"></script>
-    <uma-connect-button></uma-connect-button>
+    <uma-connect-button
+      nostr-pubkey="npub1scmpzl2ehnrtnhu289d9rfrwprau9z6ka0pmuhz6czj2ae5rpuhs2l4j9d"
```
Comment (jklein24) [inline examples/html/index.html:25]: Maybe good to call this app-identity-pubkey to be specific? Wouldn't want clients to think it's a user npub or something.

**Review by jklein24 — APPROVED** [review]
LFGGGGG

File: examples/html/index.html (hunk 1 above)
Comment (bsiaotickchong) [reply examples/html/index.html:28]: has to be a string in this example bc this is just raw html and not jsx

File: examples/html/index.html (hunk 2 above)
Comment (bsiaotickchong) [reply examples/html/index.html:25]: updated to app-identity-pubkey

**Pull request merged**